templates/docker-entrypoint.sh

```diff
@@ -150,6 +150,10 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
     echo "\$config['spellcheck_uri'] = '${ROUNDCUBEMAIL_SPELLCHECK_URI}';" >> config/config.docker.inc.php
   fi
 
+  if [[ "$ROUNDCUBEMAIL_PLUGINS" = *enigma* ]]; then
+    echo "$config['enigma_pgp_homedir'] = '/var/roundcube/enigma';" >> config/config.docker.inc.php
```

Comment:
Maybe check the setting is not already in the file?

Reply:
Do you mean `plugins/enigma/config.inc.php`? Or which file?

Reply:
I did mean `config/config.docker.inc.php` but maybe the script creates it each time